pandas/core/frame.py

```diff
@@ -2011,15 +2011,6 @@ def to_parquet(self, fname, engine='auto', compression='snappy',
         This function requires either the `fastparquet
         <https://pypi.org/project/fastparquet>`_ or `pyarrow
         <https://arrow.apache.org/docs/python/>`_ library.
-
-        Examples
-        --------
-        >>> df = pd.DataFrame(data={'col1': [1, 2], 'col2': [3, 4]})
-        >>> df.to_parquet('df.parquet.gzip', compression='gzip')
```

Comment:
This is only an attempt to get rid of #23680. If my current hypthesis (that the `ResourceWarning` is caused by the this call within the doctests) is correct, then #23680 should be solved independently and I'll revert this deletion here.